src/modules/image/index.ts

```
@@ -102,8 +102,8 @@ export class ImageModule {
    * Generates a random image url.
    *
    * @param options Options for generating a URL for an image.
-   * @param options.width The width of the image. Defaults to `640`.
-   * @param options.height The height of the image. Defaults to `480`.
+   * @param options.width The width of the image. Defaults to `this.faker.number.int({ min: 1, max: 3999 })`.
+   * @param options.height The height of the image. Defaults to `this.faker.number.int({ min: 1, max: 3999 })`.
```

I would probably refer to it as Defaults to a random value between `1` and `3999`.

(although I would include 4000 as well)

---

I saw this function and that's how I ended up doing 1 - 3999 https://fakerjs.dev/api/image.html#urlplaceholder. Ok, will change the "Defaults to" message.

---

Good catch.